src/System.Management.Automation/CoreCLR/CorePsStub.cs

```diff
@@ -477,7 +477,7 @@ internal static bool IsClassInApprovedList(Guid clsid)
 
         /// <summary>
         /// Gets the system wide script file policy enforcement for an open file.
-        /// Based on system WDAC (Windows Defender Application Control) or AppLocker policies.
+        /// Based on system WDAC (Windows Defender Application Control) policies.
```

Comment:
```suggestion
        /// Based on system WDAC (Windows Defender Application Control) or AppLocker policies.
```

Reply:
I agree to revert it back, so it's the same as the XML comment of the Windows version of this API.